Restructure Docs (#18)
CodeMC/Documentation: docs/services/ci/index.md

# CI Service
CodeMC allows you to configure steps for your project to execute whenever a new build is triggered.  

> It also has integrated support for automatic builds from various Remote Repository hosts such as GitHub, GitLab or Gitea/Forgejo.

Reviewer: Maybe just say any Git provider?
```suggestion
It also has integrated support for automatic builds from all Remote Repository hosts that support [Git](https://git-scm.com).
```
